### PR TITLE
Feature/implement benefit section

### PR DESCRIPTION
**Pull request by mthanh2209**

### CODE REVIEW

File: src/assets/css/layouts/header.css [hunk 1]
```diff
@@ -32,4 +32,8 @@
 .navbar-link {
   padding-left: 30px;
   font-weight: var(--fw-regular);
+}
+.navbar-button a{
```
Comment (phuongbuimai-agilityio) [inline src/assets/css/layouts/header.css]: - Code style
- As I told you before, we shouldn't style directly HTML elements. Please check all files and update if have

File: src/assets/css/sections/benefit_section.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
+.benefit-section {
+  height: 543px;
```
Comment (phuongbuimai-agilityio) [inline src/assets/css/sections/benefit_section.css]: Why do we need set height here?

File: src/assets/css/sections/benefit_section.css [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  margin-top: 55px;
+  margin-left: 55px;
+}
+.benefit-text {
+  text-align: center;
+  margin-left: 142px;
```
Comment (phuongbuimai-agilityio) [inline src/assets/css/sections/benefit_section.css]: Why do we need to use margin here? When should we use margin? When should we use padding?

File: src/assets/css/sections/benefit_section.css [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  margin-right: 142px;
+}
+.benefit-card {
+  display: flex;
+  margin-left: 47.5px;
+  margin-top: 74px;
+  width: 1120px;
+  height: 204px;
```
Comment (phuongbuimai-agilityio) [inline src/assets/css/sections/benefit_section.css]: Why do we need set height for each block element?

File: src/assets/css/sections/benefit_section.css [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  margin-left: 55px;
+}
+.benefit-text {
+  text-align: center;
+  margin-left: 142px;
+  margin-right: 142px;
+}
+.benefit-card {
+  display: flex;
+  margin-left: 47.5px;
```
Comment (phuongbuimai-agilityio) [inline src/assets/css/sections/benefit_section.css]: Why do we have this value?

File: src/assets/css/sections/benefit_section.css [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+  margin-top: 74px;
+  width: 1120px;
```
Comment (phuongbuimai-agilityio) [inline src/assets/css/sections/benefit_section.css]: Why do we need set width here?

File: src/assets/css/layouts/header.css [hunk 7]
```diff
@@ -1,35 +1,44 @@
 .header-section {
-  width: 100%;
-  height: 80px;
   display: flex;
   justify-content: space-between;
-  align-items: center;
+  padding-top: 25px;
+  align-items: canter;
```
Comment (hungnguyenhuu-agilityio) [inline src/assets/css/layouts/header.css]: What is canter?

File: src/assets/css/sections/benefit_section.css [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+  height: 204px;
+}
+
+.card {
+  width: 245px;
+  height: 178px;
+  margin-left: 47.5px;
```
Comment (hungnguyenhuu-agilityio) [inline src/assets/css/sections/benefit_section.css]: Magic number

File: src/assets/css/sections/benefit_section.css [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+}
+
+.card-border {
+  width: 51.142px;
```
Comment (hungnguyenhuu-agilityio) [inline src/assets/css/sections/benefit_section.css]: Magic number

File: src/assets/css/sections/benefit_section.css [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+
+.card:nth-child(4) .card-border {
+  background: #f03e3d;
+}
+
+.card-item {
+  margin-top: 30px;
+}
+
+.card-item p {
```
Comment (hungnguyenhuu-agilityio) [inline src/assets/css/sections/benefit_section.css]: should we style the element?

File: src/assets/css/sections/benefit_section.css [hunk 11]
```diff
@@ -0,0 +1,75 @@
...
+  line-height: 26px;
+}
+
+.benefit-button {
+  margin-top: 26px;
+  display: flex;
+  justify-content: center;
+}
+
+.benefit-button a {
```
Comment (hungnguyenhuu-agilityio) [inline src/assets/css/sections/benefit_section.css]: element again

File: src/assets/css/layouts/header.css (hunk 1 above)
Comment (mthanh2209) [reply src/assets/css/layouts/header.css]: Thank you, i fixed it

File: src/assets/css/layouts/header.css [hunk 12]
```diff
@@ -1,35 +1,44 @@
...
+  margin-left: 32px;
+  text-decoration: none;
   font-weight: var(--fw-regular);
+}
+
+.navbar-button {
+  padding-left: 35px;
+}
+
+.button-text {
```
Comment (hungnguyenhuu-agilityio) [inline src/assets/css/layouts/header.css:41]: This using for header only or other sections also, please help check the naming

File: src/assets/css/sections/benefit_section.css [hunk 13]
```diff
@@ -0,0 +1,59 @@
...
+  background: #f03e3d;
+}
+
+.benefit-button {
+  padding-top: 30px;
+  display: flex;
+  justify-content: center;
+}
+
+.button-title {
```
Comment (hungnguyenhuu-agilityio) [inline src/assets/css/sections/benefit_section.css:52]: The same with above comment in header

File: src/assets/css/sections/benefit_section.css (hunk 2 above)
Comment (mthanh2209) [reply src/assets/css/sections/benefit_section.css]: I misunderstood when making a section should not set the height for it

File: src/assets/css/sections/benefit_section.css (hunk 3 above)
Comment (mthanh2209) [reply src/assets/css/sections/benefit_section.css]: Sorry, I shouldn't use margin here.
1. When to use margin:
- Creating spacing between neighboring elements.
- Creating spacing between columns or boxes.
- Creating spacing between elements in a list.
- Creating spacing between an element and the borders of its parent element.

2. When to use padding:
- Creating spacing between the content and the borders of an element.
- Creating spacing between the content and the borders of the parent element.

File: src/assets/css/bases/global.css [hunk 14]
```diff
@@ -1 +1,5 @@
-/* TODO: Need to update */
+.container {
+  width: 1120px;
+  margin: 0 auto;
+  position: relative;
```
Comment (phuongbuimai-agilityio) [inline src/assets/css/bases/global.css:4]: Why do you need set position for the container?

File: src/assets/css/components/item.css [hunk 15]
```diff
@@ -0,0 +1,23 @@
...
+  align-items: flex-start;
+  gap: 10px;
+}
+
+.text-title {
+  font-size: var(--font-size-lg);
+  font-weight: var(--fw-semibold);
+}
+
+.text-desc {
```
Comment (phuongbuimai-agilityio) [inline src/assets/css/components/item.css:18]: Why do you define css for text in this file?

File: src/assets/css/utils/variables.css [hunk 16]
```diff
@@ -12,6 +12,7 @@
   --font-size-xl: 24px;
   --font-size-2xl: 32px;
 }
+
 body {
```
Comment (phuongbuimai-agilityio) [inline src/assets/css/utils/variables.css:16]: Why do you set the style for the body in this file?

File: src/index.html [hunk 17]
```diff
@@ -4,14 +4,14 @@
...
   </head>
 
   <body>
     <!-- START HEADER SECTION -->
-    <header class="header-section">
+    <header class="header-section container">
       <a href="javascript:void(0)" class="header-logo">
-        <img src="assets/images/logo.png" alt="Logo" class="logo-img" />
+        <img src="assets/images/logo.png" alt="Logo" />
         <span class="logo-text text">Product</span>
```
Comment (phuongbuimai-agilityio) [inline src/index.html:15]: Should order common class before a class of each section.

File: src/index.html [hunk 18]
```diff
@@ -29,31 +29,93 @@
             <a href="javascript:void(0)" class="navbar-link text">Resources</a>
           </li>
         </ul>
-        <a href="javascript:void(0)" class="btn btn--secondary">Sign in</a>
-        <a href="javascript:void(0)" class="btn btn--primary">Sign up</a>
+        <div class="navbar-button">
+          <a href="javascript:void(0)" class="btn btn--secondary button-text">
```
Comment (phuongbuimai-agilityio) [inline src/index.html:33]: Can we use another HTML tag for this?

File: src/index.html [hunk 19]
```diff
@@ -29,31 +29,93 @@
...
-    <div class="hero-section">
-      <div class="hero-text text">
-        <h1>Work at the speed of thought</h1>
-        <p>
+    <main class="container">
+      <!-- START HERO SECTION -->
+      <section class="hero-section text">
+        <div class="hero-content">
+          <h1 class="hero-title">Work at the speed of thought</h1>
+        <p class="hero-desc">
```
Comment (phuongbuimai-agilityio) [inline src/index.html:47]: Is tag p nested in <div class="hero-content"> ?

File: src/index.html [hunk 20]
```diff
@@ -29,31 +29,93 @@
...
+      </section>
+      <!-- END HERO SECTION -->
+
+      <!-- START BENEFIT SECTION -->
+      <section class="benefit-section">
+          <h2 class="benefit-text text">Product was Built Specifically for You</h2>
+        <div class="product-items text">
+
+          <!-- item 1 -->
+          <div class="items">
```
Comment (phuongbuimai-agilityio) [inline src/index.html:71]: Should be a singular noun

**Pull request closed without merging**